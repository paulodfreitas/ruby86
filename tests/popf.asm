.module m0
.pseg
;Teste da semantica da instrução Popf para o Y86
	irmovl $100,%esp
	irmovl $3,%eax
	pushf
	pushl %eax
	irmovl $80000000, %eax
	irmovl $ffffffff, %ebx
	addl %eax, %ebx
	popl %ecx
	popf
	halt
.end
