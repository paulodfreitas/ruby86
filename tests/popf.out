@ 0000
30
84
00
01
00
00
30
80
03
00
00
00
a0
00
a0
08
30
80
00
00
00
80
30
83
ff
ff
ff
ff
60
03
b0
08
b0
00
10
@ 0023
